app/views/links/new.html.erb

```diff
@@ -1 +1,10 @@
-<%= render 'shared/form', locals: { links: @link}%>
\ No newline at end of file
+<section class="wrapper ">
+  <div class="row mt">
+    <div class="col-lg-2 col-md-2"></div>
+    <div class="col-lg-6 col-md-6 col-sm-10 form-panel">
+      <h3> Add Link </h3><hr/>
+<%= render 'shared/form', locals: { links: @link}%>
```

Comment:
Shared folder contains templates which are used across the app. You placed form partial which is specific to Links. So it should be inside links directory only